internal/types/types.go

```diff
@@ -0,0 +1,80 @@
+package types
+
+import "time"
+
+type BucketCannedACL string
+
+// Enum values for BucketCannedACL.
+const (
+	BucketCannedACLPrivate    BucketCannedACL = "private"
+	BucketCannedACLPublicRead BucketCannedACL = "public-read"
+)
+
+func (BucketCannedACL) Values() []BucketCannedACL {
+	return []BucketCannedACL{
+		BucketCannedACLPrivate,
+		BucketCannedACLPublicRead,
+	}
+}
+
+type BucketMetadata struct {
+	Name             string         `json:"name"`
+	CacheControl     string         `json:"cache_control"`
+	ObjectRegions    string         `json:"object_regions"`
+	MD               *BucketMD      `json:"md"`
+	Shadow           *BucketShadow  `json:"shadow_bucket"`
+	Website          *BucketWebsite `json:"website"`
+	CreatedAt        time.Time      `json:"created_at"`
+	InitialCreatedAt time.Time      `json:"initial_created_at"`
```

Comment:
Both `InitialCreatedAt` and `CreatedAt` are not exposed outside.

Reply:
Ok, let me remove these fields